Source/p_mobj.h

```diff
@@ -355,6 +376,9 @@ typedef struct mobj_s
     fixed_t		oldy;
     fixed_t		oldz;
     angle_t		oldangle;
+
+    // mbf21
+    int flags2;
```

Comment:
MBF21 added new mobj flags, so we need to add a new field to `mobj_t` and broke saves. Maybe we should introduce some kind of versioning?

Reply:
Yes, we should replace `VERSIONID` with a versioned Woof-specific identifier each time we break savegame compatibility.

Reply:
We aren't MBF-compatible anymore for quite some time anyway.

Reply:
Should we try to support the previous version of saves?

Reply:
Yes, I think this would be only fair.